README.md

```diff
@@ -196,3 +205,74 @@ operator-sdk cleanup packagemanifests --olm-namespace openshift-operator-lifecyc
 
 *Operator-sdk has a known bug while using `operator-sdk run/cleanup --olm` where it shows failure on success. 
 Track the issue [here](https://github.com/operator-framework/operator-sdk/issues/2938). The error does not imply that the operator will not work.*
+
+### Creating a Windows MachineSet for testing
```

Comment:
nit: Should `for testing` be here? Yes it can be used for testing, but also for actual use :)